images/installPlatform.sh

```diff
@@ -51,18 +52,22 @@ if [ -z "$sdkStorageAccountUrl" ]; then
 fi
 if [ "$sdkStorageAccountUrl" == "$PRIVATE_STAGING_SDK_STORAGE_BASE_URL" ]; then
     set +x
-    isSasTokenEmpty=1 
-    sasToken=$ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN
+
+    if [ "$IS_FROM_DYNAMIC_INSTALLATION" = "true" ]; then
```

Comment:
nit: use booleans instead of string?

Reply:
im fine with this for now, but i do believe that there are booleans in bash: https://stackoverflow.com/questions/2953646/how-can-i-declare-and-use-boolean-variables-in-a-shell-script/21210966